gossip/src/node.rs

```diff
@@ -117,10 +126,7 @@ impl Node {
             )
             .expect("Secondary bind TVU"),
         );
-        let tvu_addresses: Vec<SocketAddr> = bind_ip_addrs
-            .iter()
-            .map(|&ip| SocketAddr::new(ip, tvu_port))
-            .collect();
+        let tvu_addresses = Self::get_socket_addrs(&bind_ip_addrs, tvu_port);
```

Comment:
Nit:
Would it not be preferred to generate these from actually bound sockets to avoid desync?